src/webgpu/api/validation/image_copy/layout_related.spec.ts

```diff
@@ -191,13 +191,17 @@ Test the computation of requiredBytesInCopy by computing the minimum data size f
     const bytesPerRowAlignment = method === 'WriteTexture' ? 1 : 256;
     const copyWidth = copyWidthInBlocks * info.blockWidth;
     const copyHeight = copyHeightInBlocks * info.blockHeight;
-    const offset = offsetInBlocks * info.bytesPerBlock;
     const rowsPerImage = copyHeight + rowsPerImagePaddingInBlocks * info.blockHeight;
     const bytesPerRow =
       align(bytesInACompleteRow(copyWidth, format), bytesPerRowAlignment) +
       bytesPerRowPadding * bytesPerRowAlignment;
     const copySize = { width: copyWidth, height: copyHeight, depthOrArrayLayers: copyDepth };
 
+    let offset = offsetInBlocks * info.bytesPerBlock;
+    if (kTextureFormatInfo[format].depth || kTextureFormatInfo[format].stencil) {
+      offset = offsetInBlocks * 4;
```

Comment:
hm.. now the `offsetInBlocks` param doesn't make any sense for stencil8 and depth16unorm.

maybe instead in the params we should compute it based on the format inside a `.expand(...)`